api/signals.py

```diff
@@ -30,3 +42,61 @@ def track_application_actions(sender, instance, **kwargs):
             history.save()
         except ValueError:
             pass
+
+
+@receiver(post_save, sender=AssetIncidentReport)
+def create_notification_on_incident_report_submission(**kwargs):
+    """
+    Create a notification to admin users when a user creates an incident report
+    :param kwargs:
+    :return:
+    """
+    instance = kwargs["instance"]
+    # if a new notification has been  created
+    if kwargs.get("created", False) is True:
+        notification_body = "{} submitted a new incident report {}."
```

Comment:
@fgkinus good work, I have suggestions. the notification details could be added in the constant.py file. we would just call them from there then format them if we need to add contents to them. I mean the notification body, title. we could also do away with line 60 and just add the instance.submitted_by directly to origin on line 69.

Reply:
Thanks for these notes. I agree and  will incorporate the feedback 👍